db/seeds.d/115-http_proxy.rb

```diff
@@ -0,0 +1,25 @@
+if (SETTINGS[:katello][:cdn_proxy] && SETTINGS[:katello][:cdn_proxy][:host])
+  config = SETTINGS[:katello][:cdn_proxy]
+  uri = URI(config[:host])
```

Comment:
When I am specifying the cdn setting in the katello config, the host is not picked up unless I format the string as a URL (including the HTTP scheme). For example, "http://centos7-katello-devel.example.com/".

Is this the real meaning of the :cdn_proxy, :host entry? I was expecting either an IP address or a hostname to work here.

Reply:
the install does require a full uri "http://foo.com", and not just a hostname, this is to indicate whether http or https is being used.  
example:
```
# foreman-installer  --katello-proxy-url=redhat.com
Resetting puppet server version param...
Parameter katello-proxy-url invalid: redhat.com must match one of (?i-mx:^https?:\/\/)
```

Reply:
Ok, so it looks like this is the established convention. Thanks!